gtsam/linear/tests/testNoiseModel.cpp

```diff
@@ -452,19 +452,24 @@ TEST(NoiseModel, WhitenInPlace)
 /*
  * These tests are responsible for testing the weight functions for the m-estimators in GTSAM.
  * The weight function is related to the analytic derivative of the residual function. See
- *  http://research.microsoft.com/en-us/um/people/zhang/INRIA/Publis/Tutorial-Estim/node24.html
+ *  https://members.loria.fr/MOBerger/Enseignement/Master2/Documents/ZhangIVC-97-01.pdf
```

Comment:
Changes in this file (except std::abs) belong in other PR

Reply:
I just force-pushed so the PRs should now be truly separate.